tests/test_snapcraft_store_api.py

```diff
@@ -42,10 +42,10 @@ def test_search_by_arch(self):
         result_wide = self.client.search("toto", 10, 1, arch="wide")
         result_i386 = self.client.search("toto", 10, 1, arch="i386")
 
-        self.assertEqual(result_amd64["total"], 6)
-        self.assertEqual(result_default["total"], 7)
-        self.assertEqual(result_wide["total"], 7)
-        self.assertEqual(result_i386["total"], 3)
+        self.assertEqual(result_amd64["total"], 5)
```

Comment:
We shouldn't be searching for snaps called "toto" to test the search method because the number of results are going to be continually changing.

Reply:
I created https://github.com/canonical-web-and-design/canonicalwebteam.store-api/issues/26

Reply:
@jkfran is there no way to test this without using a hard-coded number? Surely any search at all might return a changing number of results. But if there is no way, then it doesn't really matter, because we presumably testing against a fixture. I think it's fine to write the test against the contents of the fixture, and then if the fixture changes it's reasonable to expect that the test might have to change.

Reply:
@nottrobin yeah, I agree that if it is no big deal if we don't update the fixtures but anyway we need to solve this issue because we do a daily test against the real API and it is also failing because of this.

Reply:
oh ok. For that test, I don't think we can expect a hard-coded number of results at all.